src/scene/vcPOI.cpp

```diff
@@ -1033,16 +1033,16 @@ void vcPOI::Cleanup(vcState *pProgramState)
   vcLineRenderer_DestroyLine(&m_pLine);
   udFree(m_pLabelInfo);
 
-  if (pProgramState->cameraInput.pAttachedToSceneItem == this)
-    pProgramState->cameraInput.pAttachedToSceneItem = nullptr;
+  if (pProgramState->pActiveViewport->cameraInput.pAttachedToSceneItem == this)
+    pProgramState->pActiveViewport->cameraInput.pAttachedToSceneItem = nullptr;
 }
 
 void vcPOI::SetCameraPosition(vcState *pProgramState)
 {
   if (m_attachment.pModel)
-    pProgramState->camera.position = m_attachment.currentPos;
+    pProgramState->pActiveViewport->camera.position = m_attachment.currentPos;
   else
-    pProgramState->camera.position = m_pLabelInfo->worldPosition;
+    pProgramState->pActiveViewport->camera.position = m_pLabelInfo->worldPosition;
```

Comment:
The base function from `vcSceneItem` sets all of the viewports, not just the active one.